### ISSUE TITLE
Add keyboard navigation to SelectableListBoxDropdown

### ISSUE THREAD

**Issue by ashklianko** [enonic/lib-admin-ui#2612]
@sgauruseu commented on [Wed Jun 01 2022](https://github.com/enonic/app-contentstudio/issues/4622)

1. Type an user name in "Last modified by" options filter input

**Issue**:  keyboard navigation doesn't work in the selector:

![image](https://user-images.githubusercontent.com/3728712/171348583-05bf6f18-b02c-4ab4-8336-58b2c9d2e37b.png)
